projects/stone_cx/simulator.cc

```diff
@@ -13,7 +13,9 @@
 // Common includes
 #include "plog/Log.h"
 #include "common/von_mises_distribution.h"
-#include "genn_utils/analogue_csv_recorder.h"
+
+// GeNN userproject includes
+#include "analogueRecorder.h"
```

Comment:
This is a bit off-topic, but maybe it would make more sense to have these headers in ``genn/utils`` or something instead of with the userprojects.

Reply:
that would probably be nicer but probably not enough to break backward compatibility